### PR TITLE
Enforce opinionated commit messages

### LINKED ISSUE #63: Consider using opinionated-commit-message
This is a very minor thing to consider, but helps with a clean commit history: https://github.com/mristin/opinionated-commit-message

Please let me know if I should add it to the GitHub workflows.

### PR DESCRIPTION
**Pull request by mristin**
Fixes #63.

### CODE REVIEW

File: .github/workflows/check-pull-request-title-and-description.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      - name: Check the pull request
+        uses: mristin/opinionated-commit-message@v2.3.1
+        with:
+          path-to-additional-verbs: additional-verbs-in-imperative-mood.txt
+
+      - name: READ HERE ON FAILURE FOR MORE INSTRUCTIONS
+        if: ${{ failure() }}
+        run: |
+          echo \
+            "The title and the description of your pull request do not fit our style guide." \
```
Comment (pschanely) [inline .github/workflows/check-pull-request-title-and-description.yml:21]: Maybe we don't talk about a style guide until we really add one. WDYT?

**Review by pschanely — CHANGES_REQUESTED** [review]

File: .github/workflows/check-pull-request-title-and-description.yml (hunk 1 above)
Comment (mristin) [reply .github/workflows/check-pull-request-title-and-description.yml:21]: Thanks for spotting it! I copy-pasted the script and adapted it. Though I removed the details from the other project, I did not pay attention to the actual message.

Yes, I agree, let's wait for the guidelines to be defined. The guideline should also be linked in the message.

**Comment by mristin** [comment]
The guideline should be linked once #60 is fixed.

**Comment by mristin** [comment]
Note to myself: reference this check in the contributing guidelines once the pull request #71 is merged in.

**Pull request merged**